src/lib.rs

```diff
@@ -64,6 +64,7 @@ mod internal_macros;
 #[macro_use]
 pub mod consensus;
 pub mod blockdata;
+pub mod bulletproof;
```

Comment:
Can we make the module private and only export directly what we need?

At the moment, the compiler will have a hard time flagging unused code because a lot of things are `pub`.